DatadogRUM/Sources/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -36,6 +36,8 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
     let identity: ViewIdentifier
     /// View attributes.
     private(set) var attributes: [AttributeKey: AttributeValue] = [:]
+    /// Internal view attributes - used by cross platform frameworks and should not be propoagated to events
```

Comment:
```suggestion
    /// Internal view attributes - used by cross platform frameworks and should not be propagated to events
```
Just a little typo 😄